.gitignore

```diff
@@ -176,3 +176,8 @@ notebooks/*.yaml
 .ipynb_checkpoints/
 
 !configs/*.yaml
+
+
+# Google API
+credentials.json
+token.json
```

Comment:
Hmm I think these shouldn't be here in the repo. User should keep these somewhere else.

Reply:
Yes, the requirement is it should be in the directory where they run the loader. So will update the doc to mention this pre-req and remove from .gitignore, thanks!